src/Solver.h

```diff
@@ -126,9 +126,9 @@ class Solver {
   /** The total number of Tracks */
   int _tot_num_tracks;
 
-  /** An integer array with the Track uid separating the azimuthal and periodic
-   * halfspaces */
-  int* _num_tracks_by_halfspace;
+  /** An integer array with the Track uid separating the parallel track
```

Comment:
That makes perfect sense, and I realized it myself but only after digging through some of the other code to figure out how the UIDs are set. It's not a big deal, just wasn't exactly clear what was meant by the comment at first sight.